clients/src/main/resources/common/message/README.md

```diff
@@ -85,6 +87,8 @@ There are several primitive field types available.
 
 * "bytes": binary data.
 
+* "records": record set used in fetch api and fetch snapshot api
```

Comment:
`records` is used by produce data also.

Reply:
Thank you, I think it's better to remove usage cases or we need to update docs every time we use it, I reworded this doc, PTAL.